src/routes/+page.svelte

```diff
@@ -13,121 +14,49 @@
 	<title>Dashboard The Ocean Cleanup</title>
 </svelte:head>
 
+<Navigation />
+
 <section class="main">
-	<div class="container2">
+	<div class="grid-container">
+
 		<!-- Blue line -->
 		<div class="menu">
-			<div class="line" />
+			<div class="blue-dashboard-line" />
```

Comment:
Dit is misschien een risky classname omdat we niet weten of deze altijd blauw zal zijn?

Reply:
Heb het nu dashboard-line genoemd :)